src/app/utils/plug-descriptions.ts

```diff
@@ -204,6 +205,14 @@ function getPerkDescriptions(
       usedStrings.add(notif);
     }
   }
+  function addSpecialDescriptions() {
+    if (plug.displayProperties.name.includes('Harmonic')) {
```

Comment:
This will only work when DIM is set to English. We'd have to find a different way to identify them, perhaps by generating a list in https://github.com/DestinyItemManager/d2-additional-info.

Reply:
ah thats my bad when I was making these changes I was thinking about adding harmonic to the translation list but I wasn't sure how just the word "harmonic' would translate if it doesn't translate well then adding to d2ai may be the best option.

Reply:
Resolved by d2ai PR